hash/spectrum_cass.xml

```diff
@@ -147145,11 +148034,11 @@ Version without loading screen, maybe a pre-release
 
 	<software name="vegasolrg" cloneof="vegasol">
 		<!--
-		    Identical to the (previously recovered) unreleased final Spanish version, freebie for Spanish Retro Gamer subscribers.
+		    Identical to the (previously recovered) unreleased final Spanish version, freebie for Spanish RetroGamer subscribers.
 		-->
-		<description>Vega Solaris (Retro Gamer)</description>
+		<description>Vega Solaris (RetroGamer)</description>
```

Comment:
Where are you getting “RetroGamer” as one word from?  Officially it seems to be two words.

It’s usually written as two words on two line in box art, etc: https://www.hobbyconsolas.com/reviews/analisis-vega-solaris-para-zx-spectrum-57994

Official store pages had more instances of “Retro Gamer” than the rare instances of “RetroGamer”: https://web.archive.org/web/20130920055039/http://store.axelspringer.es/videojuegos/revistas-videojuegos/retrogamer/pack1-retrogamer.html